dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.adminSecurity-2.0.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.adminSecurity-2.0
```

Comment:
io.openliberty....

Reply:
This cannot happen because we need to tolerate across the version of the private feature in `restHandler-1.0`

If we really want io.openliberty used here then we should do a concerted effort to rename ALL of our private features to io.openliberty...

Reply:
Keeping it the same for now.